packages/dds/merge-tree/src/collections.ts

```diff
@@ -144,13 +144,6 @@ export class List<T> {
         return (this.next === this);
     }
 
-    /**
-     * @deprecated - use unshift
-     */
-    public push(data: T): void {
-        this.unshift(data);
-    }
-
```

Comment:
Same here, separate callout needed in breaking changes? Or since merge-tree is not GA yet, we're not pointing those out as strictly?

Reply:
Agreed if they're internal... however, in both cases they seem to be exported out of the package?

In merge-tree `index.ts` does `export * from "./collections";`. And in sequence, it does `export { ..., SequenceInterval,
} from "./intervalCollection";`. Should they have at least an `@internal` TSDoc annotation?

Reply:
yeah. unfortunately, our exports are not as clean as we'd like, and not everything has been tagged for visibility. i did do the appropriate deprecation flow here for extra safety, but i don't expect anyone to be using this

Reply:
this is also the next branch, so breaks are allowed here